Resources/Prototypes/DeltaV/Entities/Mobs/Player/silicon.yml

```diff
@@ -30,4 +30,4 @@
         name: power-cell-slot-component-slot-name-default
         startingItem: PowerCellMedium
   - type: RandomMetadata
-    nameSegments: [names_borg] # TODO: Make good names, this is a stupid list.
+    nameSegments: [names_borg_deltav] # TODO: Make good names, this is a stupid list. # Your wish is my command, unknown commenter -Fox
```

Comment:
just remove the comment so it wont show on TODOs

Reply:
the comment intent was to have secborg with less goofy names, not that this pr is bad just saying

also remove the comments yes